pkgs/development/compilers/chicken/4/chicken.nix

```diff
@@ -21,24 +21,32 @@ stdenv.mkDerivation {
     sha256 = "0hvckhi5gfny3mlva6d7y9pmx7cbwvq0r7mk11k3sdiik9hlkmdd";
   };
 
-  setupHook = lib.ifEnable (bootstrap-chicken != null) ./setup-hook.sh;
+  setupHook = lib.optional (bootstrap-chicken != null) ./setup-hook.sh;
 
-  buildFlags = [ "PLATFORM=${platform}" "PREFIX=$(out)" "VARDIR=$(out)/var/lib" ];
```

Comment:
There is a reference to `$buildFlags` in the call to `make spotless` below, it breaks the build of `chickenPackages_4.chicken`.